.github/workflows/pytest.yml

```diff
@@ -68,6 +68,13 @@ jobs:
           mc admin policy attach local readwrite --user=mynewuser
           mc admin accesskey create local mynewuser --access-key myuseraccesskey --secret-key myusersecretkey
 
+      - name: "Patch mock_awx postgres config"
+        run: |
+          sed -i '/USER/s/myuser/awx/' mock_awx/settings/__init__.py
+          sed -i '/PASSWORD/s/mypassword/awx/' mock_awx/settings/__init__.py
+          echo \\l | sudo su - postgres -c psql
+          echo \\dt | sudo su - postgres -c psql
+
```

Comment:
I guess its unclear to me why we need this.

Reply:
It should eventually be configurable, but right now, the DB test gets the DB credetnials from that file.

But those are different when using the docker compose env vs github CI, so this patches the config to match reality.